app/views/duties/index.html.erb

```diff
@@ -67,9 +67,21 @@
                     <% place.each do |duty| %>
                       <% if duty[:name].nil? %>
                         <td class="schedule-name-empty" colspan="<%= duty[:colspan] %>">&nbsp;</td>
+                      <% elsif duty[:free] || current_user.id == duty[:ruid] %>
+                        <td class="schedule-name-empty text-center" colspan="<%= duty[:colspan] %>">
+                           <%= button_to "Grab", {controller: "duties", action: "open_grab_modal", grab_duty_list: duty[:span_duty]}, {method: :post, remote: true, class: "btn btn-link", 'data-toggle': "modal", 'data-target': '#modal-window1'} %>
```

Comment:
Mind the hash spacing and single quotes: `{ controller: 'duties', action: 'open_grab_modal', ... }`